src/Integration.UnitTests/LocalServices/BindingCheckerTests.cs

```diff
@@ -29,7 +29,28 @@ namespace SonarLint.VisualStudio.Integration.UnitTests.LocalServices
     public class BindingCheckerTests
     {
         [TestMethod]
-        public void IsBindingUpdateRequired_NoUnboundProjects_False()
+        public void IsBindingUpdateRequired_SolutionIsUnbound_True()
+        {
+            var testSubject = CreateTestSubject(isSolutionBound: false);
+
+            var result = testSubject.IsBindingUpdateRequired();
+
+            result.Should().BeTrue();
```

Comment:
Perhaps check that the unbound solution checker is called in this test and the next one?